packages/core/src/lib/markdown-it/highlight/HighlightRule.js

```diff
@@ -1,6 +1,8 @@
 const { HighlightRuleComponent } = require('./HighlightRuleComponent');
 const { splitCodeAndIndentation } = require('./helper');
 
+const HIGHLIGHT_LINES_RULES_DELIMITER_REGEX = /(?<!'.*)-|-(?!.*')/;
```

Comment:
Since the other regex-based highlighting rules are defined in `HighlightRuleComponent.js` as seen [here](https://github.com/MarkBind/markbind/blob/master/packages/core/src/lib/markdown-it/highlight/HighlightRuleComponent.js#L3-L6), would it make more sense to declare it there instead of here? Just a suggestion!

Reply:
Hmmm I'm not too sure about this. Because the ```HighlightRuleComponent``` is about parsing the components of the rules. So the splitting of the components should be done before the components are made.

Reply:
Alright, that makes sense. In that case I have no further issues :+1: